### PR TITLE
Support for MultiQC NGI module

### PR DESCRIPTION
**Pull request by sylvinite**
Added support for MultiQC NGI module.
Trimmed the code in some places.

### CODE REVIEW

File: roles/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+multiqc_ngi_repo: https://github.com/ewels/MultiQC_NGI.git
```
Comment (johanherman) [inline roles/defaults/main.yml]: I'm guessing you want to have this in `roles/multiqc/defaults/main.yml` :)

File: roles/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+multiqc_ngi_dest: "{{ root_path }}/sw/multiqc_ngi"
```
Comment (johanherman) [inline roles/defaults/main.yml]: If the `multiqc_ngi` dir only exists as a placeholder for your checkout, then perhaps you could rename this to `multiqc_ngi_src` or something similar, to make it more clear that no sw is actually there. Perhaps we should even start using a `/lupus/ngi/src/` instead in the future.

File: roles/multiqc/tasks/main.yml [hunk 3]
```diff
@@ -4,15 +4,29 @@
   shell: "{{ ngi_pipeline_venv }}/bin/pip install multiqc --upgrade"
```
Comment (johanherman) [inline roles/multiqc/tasks/main.yml]: Should multiqc from pypi be installed at the same time as multiqc from the github repo?

File: roles/defaults/main.yml (hunk 1 above)
Comment (sylvinite) [inline roles/defaults/main.yml]: Woops

File: roles/multiqc/tasks/main.yml (hunk 3 above)
Comment (sylvinite) [inline roles/multiqc/tasks/main.yml]: Yes. "MultiQC is from github" is actually a multiqc plugin.

**Pull request merged**